nautobot/core/ui/echarts.py

```diff
@@ -245,7 +245,7 @@ def __init__(
                     }
                 You can also hide the legend entirely by setting `"show": False`.
                 More details here: https://echarts.apache.org/handbook/en/concepts/legend.
-            theme (str): Theme for chart (default: EChartsTypeTheme.DEFAULT).
+            theme_colors (str): Color palette for chart (default: EChartsThemeColors.DEFAULT).
```

Comment:
This should be a list of dictionaries, correct? We should describe the dictionary as well.